src/integrationTest/java/uk/gov/moj/sdt/controllers/FeedbackControllerIT.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.moj.sdt.controllers;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import uk.gov.moj.sdt.base.WireMockBaseTest;
+import uk.gov.moj.sdt.request.CMCFeedback;
+
+import java.util.Calendar;
+import javax.inject.Inject;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class FeedbackControllerIT extends WireMockBaseTest {
```

Comment:
Should this test class use package protection (by removing public modifier from class and test methods)?

Reply:
removed public